python/astro_metadata_translator/translators/decam.py

```diff
@@ -393,6 +393,35 @@ def fix_header(
             modified = True
             log.debug("%s: Set FILTER to %s because OBSTYPE is %s", log_label, header["FILTER"], obstype)
 
+        # The following provides corrections to the boresight header info for
+        # the raw data files obtained from http://astroarchive.noirlab.edu/,
+        # program number 2013A-0351, of the Trifid and Lagoon Nebulae region.
+        if "Trifid" in header.get("OBJECT", "unknown") and obstype == "object":
+            date_obs = astropy.time.Time(header["DATE-OBS"], format="isot", scale="utc")
+            date_min = astropy.time.Time("2013-02-10", format="isot", scale="utc")
```

Comment:
In LATISS things like these are pre-calculated on import rather than calculated per-header fixup. It might not make much of a difference here because the if-block is gated by the OBJECT. In LATISS the data checks happen for every file without gating.